cedar-policy-cli/src/lib.rs

```diff
@@ -94,10 +97,34 @@ pub enum Commands {
     Link(LinkArgs),
     /// Format a policy set
     Format(FormatArgs),
+    /// Translate JSON schema to custom schema syntax and vice versa (except comments)
+    TranslateSchema(TranslateSchemaArgs),
     /// Create a Cedar project
     New(NewArgs),
 }
 
+#[derive(Args, Debug)]
+pub struct TranslateSchemaArgs {
+    #[arg(long)]
+    pub from_json: bool,
+    #[arg(short, long = "schema", value_name = "FILE")]
+    pub input_file: Option<String>,
```

Comment:
Doc comments here would be helpful, would appear in the CLI `help`.

I wonder if we want an enum `"json-to-human", "human-to-json"` instead of a `bool` flag.  Or even, two separate CLI commands, one for each direction. so `cedar schema-json-to-human` and `cedar schema-human-to-json` or something